src/apps/handlers/prepareMovementData.js

```diff
@@ -1,10 +1,21 @@
 export default function getMovementData(data) {
   // eslint-disable-next-line no-unused-vars
-  const speeds = Object.entries(data.system.attributes.movement).filter(([_, speed]) => speed);
+  const hover = data.system.attributes.movement.traits.hover
+  const speeds = Object.entries(data.system.attributes.movement).filter(([key, speed]) => {
+    if (key === "fly" && !speed && hover) { return true; }
+    if (key === "traits") { return false; }
+    return speed
+  })
+
   const unit = game.i18n.localize('A5E.MeasurementFeetAbbr');
+  const hoverAbbr = game.i18n.localize('A5E.MovementHoverAbbr');
 
   return speeds.map(([name, speed]) => {
```

Comment:
I don't like the use of `name` here. `movementType` would be more appropriate, I think. Much more explicit.

Reply:
It looks like we use `mode` and `distance` in the `MovementConfigDialog`. Regardless of what names we choose, I'd like this to be consistent across these files.